hadoop-common-project/hadoop-common/src/site/markdown/filesystem/filesystem.md

```diff
@@ -1071,6 +1074,8 @@ entry-by-entry delete operation.
 This can break the expectations of client applications for O(1) atomic directory
 deletion, preventing the stores' use as drop-in replacements for HDFS.
 
+* [The feature of protected directories](#ProtectedDirectories) can be supported.
```

Comment:
@steveloughran what's your take? I thought this feature only concerns HDFS. Feels awkward to me that we talk about this behavior in a file system specification. (To me this is an implementation detail)